docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md

```diff
@@ -1,196 +1,229 @@
 
+  - Virtual CPU (vCPU)
+  - VNet
+  - Static Public IP addresses
+  - Virtual Network Interfaces
+  - Load Balancers
+  - Virtual Hard Disk (VHD)
+  - Managed Disks
+  - Virtual Network Address Translation (NAT) Gateway
 
-2. In the cluster page click **+ Add New Cluster** button and select **create new cluster**.
+<!-- <video title="azure-cluster-creation" src="/videos/clusters/public-cloud/azure/azure.mp4"></video> -->
 
+## Deploy an Azure Cluster
 
-3. Select **Azure** as the cloud type and click on **Start Azure Configuration** to input cluster information
+Use the following steps to deploy an Azure cluster.
 
+1. Log in to [Palette](https://console.spectrocloud.com). 
 
-4. Provide the basic cluster information such as **Name**, **Description** (optional), and **Tags** (optional) and select the [**Azure Cloud Account**](azure-cloud.md#enable-azure-cloud-account-registration-to-palette) from the drop-down menu. Azure cloud accounts with credentials must be pre-configured in project settings. Click on the **Next** button.
+2. Ensure you are in the correct project scope.
 
+3. From the left **Main Menu** select **Clusters**, and click on the **Add New Cluster** button.
 
-5. Select the **Cluster Profile** created for the Azure environment. The profile definition will be used as the cluster construction template. Click on **Next**.
+4. Select **Deploy New Cluster** on the next page that Palette displays. This will allow you to deploy a cluster using your own cloud account.
 
+5. Select **Azure** and click on the **Start Azure Configuration** button.
 
-6. Review and override pack parameters as desired. By default, parameters for all packs are set with values defined in the Cluster Profile. Click on **Next**.
+6. Fill out the following basic information, and click **Next** to continue.
 
+  | **Field** | **Description** |
+  |-----------|-----------------|
+  | **Cluster Name**| A custom name for the cluster. |
+  | **Description**| Use the description to provide context about the cluster.|
+  | **Tags**| Assign any desired cluster tags. Tags on a cluster are propagated to the Virtual Machines (VMs) deployed to the target environments. Example: `region:france south`|
+  | **Cloud Account** | If you already added your Azure account in Palette, select it from the **drop-down Menu**. Otherwise, click on **Add New Account** and add your Azure account information. |
 
-7. Provide the Azure Cloud account placement information for cluster configuration. If you have custom storage accounts or storage container available, they will be eligible for attachment. To learn more about attaching custom storage to a cluster, check out the [Azure storage](architecture#azure-storage) page.
+  To learn how to add an Azure account, review the [Register and Manage Azure Cloud Account](azure-cloud.md) guide.

```

Comment:
This was called out in the prereqs so we can probably do without it here. Or, add it to the **Cloud Accounts** description.

Reply:
Removed